src/main/java/com/minecolonies/coremod/colony/requestsystem/management/handlers/ResolverHandler.java

```diff
@@ -53,7 +53,7 @@
      * @throws IllegalArgumentException is thrown when either the token attached to the resolver is already registered or the resolver is already registered with a different
      *                                  token
      */
-    public static IToken<?> registerResolver(final IStandardRequestManager manager, final IRequestResolver resolver)
+    public static IToken<?> registerResolver(final IStandardRequestManager manager, final IRequestResolver<?> resolver)
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Remove usage of generic wildcard type. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orionminecraft.com/coding_rules#rule_key=squid%3AS1452)